### PR TITLE
libdokan: Use custom Open/Create in tests when sharing file handles o…

### PR DESCRIPTION
**Pull request by taruti**
…ver renames

For https://github.com/keybase/kbfs/pull/1092

### CODE REVIEW

File: libdokan/createfile_test.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+)
+
+func Open(filename string) (*os.File, error) {
+	return OpenFile(filename, os.O_RDONLY, 0666)
+}
+func Create(filename string) (*os.File, error) {
+	return OpenFile(filename, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0666)
+}
+
+func OpenFile(filename string, mode, perm int) (*os.File, error) {
```
Comment (strib) [inline libdokan/createfile_test.go]: Can you please put this function above `Open` for readability?

File: libdokan/createfile_test.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
+package libdokan
+
+import (
+	"os"
+	"syscall"
+)
+
+func Open(filename string) (*os.File, error) {
```
Comment (strib) [inline libdokan/createfile_test.go]: These functions can be lower cased, but I don't care much.

File: libdokan/createfile_test.go [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+	h, err := syscall.CreateFile(path, access,
+		syscall.FILE_SHARE_READ|syscall.FILE_SHARE_WRITE|syscall.FILE_SHARE_DELETE,
+		nil, create, syscall.FILE_ATTRIBUTE_NORMAL, 0)
+	if err != nil {
+		return nil, err
+	}
+	return os.NewFile(uintptr(h), filename), nil
+}
+
+func isSet(bit, value int) bool {
```
Comment (strib) [inline libdokan/createfile_test.go]: And please put this above `OpenFile`.

File: libdokan/createfile_test.go (hunk 2 above)
Comment (taruti) [inline libdokan/createfile_test.go]: They are 1:1 to os package functions, thus want to keep the names the same. As this is a _test file they won't be exported.

File: libdokan/createfile_test.go (hunk 1 above)
Comment (taruti) [inline libdokan/createfile_test.go]: Sure, moved the functions around.

File: libdokan/createfile_test.go (hunk 3 above)
Comment (taruti) [inline libdokan/createfile_test.go]: Done.

**Pull request merged**